xero-app-store.yml

```diff
@@ -0,0 +1,260 @@
+openapi: 3.0.0
+info:
+  version: "2.14.0"
+  title: Xero AppStore API
+  description: These endpoints are for Xero Partners to interact with the App Store Billing platform
+  termsOfService: "https://developer.xero.com/xero-developer-platform-terms-conditions/"
+  contact:
+    name: "Xero Platform Team"
+    email: "api@xero.com"
+    url: "https://developer.xero.com"
+  license:
+    name: MIT
+    url: "https://github.com/XeroAPI/Xero-OpenAPI/blob/master/LICENSE"
+servers:
+  - description: Xero App Store API
+    url: "https://api.xero.com/appstore/2.0/"
+paths:
+  "/subscriptions/{subscriptionId}":
+    get:
+      tags:
+        - AppStore
+      operationId: getSubscription
+      summary: Retrieves a subscription for a given subscriptionId
+      parameters:
+        - $ref: "#/components/parameters/subscriptionId"
+      responses:
+        "200":
+          description: Success - return response of unique Subscription object
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Subscription"
+              example: '
+                        {
+                          "currentPeriodEnd": "2021-07-20T03:13:48Z",
+                          "endDate": "2021-07-20T03:13:48Z",
+                          "id": "01b5a6f4-8936-4bfa-b703-830702312b87",
+                          "isTestMode": true,
```

Comment:
we should confirm if it's `isTestMode` or `testMode` and update the docs example to match

Reply:
![Screen Shot 2021-08-05 at 1 49 51 PM](https://user-images.githubusercontent.com/2935387/128413596-6cc47fe9-eca2-4883-b1cd-2ec27303d4f5.png)

Json coming back from API is `testMode`

Reply:
sounds like we confirmed this should be `testMode` in the example block instead of `isTestMode`